client/stylesheets/app.scss

```diff
@@ -1,20 +1,12 @@
-.app-content-fullscreen {
+.connection-status {
   position: fixed;
   top: 50px;
-  bottom: 0px;
   left: 0px;
   right: 0px;
+  z-index: 10000;
 }
 
-.connection-status-fullscreen {
-  position: fixed;
-  top: 50px;
-  left: 0px;
-  right: 0px;
-  z-index: 1;
-}
-
-.app-content-scrollable {
+body {
```

Comment:
Putting this on `<body>` in the middle of `app.scss` still strikes me as odd, but I'll live

but maybe this belongs in a different file?

Reply:
It's possible we could put this on the `.jolly-roger` div, but I'd want to re-test it to make sure it works consistently

Reply:
This works on the `.jolly-roger` div, so that's a bit cleaner